src/ui/controllers/course/index.ts

```diff
@@ -55,14 +53,10 @@ function getCourseDetails(
 	]
 }
 
-function getTagValues(
-	req: extended.CourseRequest,
-	tagName: string,
-	tags: string[]
-) {
+function getTagValues(tagName: string, tags: string[]) {
 	return tags
 		.filter(tag => tag.startsWith(tagName))
-		.map(tag => req.__(tag.replace(`${tagName}:`, '')))
+		.map(tag => i18n.__(tag.replace(`${tagName}:`, '')))
```

Comment:
Probably a rebase issue - `req.__` is the right way to do this (uses the users locale)

Reply:
using current master's version of this func now